src/Checker/OpenSourceVulnerabilitiesChecker/OpenSourceVulnerabilitiesChecker.php

```diff
@@ -77,22 +71,25 @@ public function getName(): string
      */
     public function check(CheckerInputDataDto $inputData): CheckerResponseDto
     {
-        $securityOutput = new BufferedOutput();
-        $this->application->setAutoExit(false);
-        $this->application->run(
-            new ArrayInput([
-                'command' => static::COMMAND_NAME,
-                '--path' => $this->pathResolver->getProjectDir(),
-                '--format' => 'json',
-            ]),
-            $securityOutput,
-        );
-
-        $rawViolations = $securityOutput->fetch();
-
-        $violations = json_decode($rawViolations, true);
-
-        if (!is_array($violations)) {
+        $process = new Process([
```

Comment:
don't use the new process. only application

Reply:
But application can execute only Symfony Console commands